Resources/Prototypes/Entities/Structures/Power/chargers.yml

```diff
@@ -93,10 +98,76 @@
 - type: entity
   name: wall recharger
   id: WallWeaponCapacitorRecharger
-  parent: WeaponCapacitorRecharger
+  placement:
+    mode: SnapgridCenter
   components:
   - type: Sprite
+    netsync: false
```

Comment:
Sprite component is client only so no need for netsync: false
https://discord.com/channels/310555209753690112/928056457117696020/1102795784673579149